src/cljs/athens/views/node_page.cljs

```diff
@@ -218,7 +223,11 @@
              :label [:> mui-icons/ExpandMore]
              :style page-menu-toggle-style}]
     (when @show-page-menu?
-      [page-menu-component {:style {:position "fixed" :left (str (:x @page-menu-position) "px") :top (str (:y @page-menu-position) "px")}}])
+      [page-menu-component {:style {:position "fixed"
```

Comment:
Where were we on the interface to our components? Passing a style map 2-3 levels here.

Reply:
Here: #278